### ISSUE TITLE
AzureDevops Plugin

### ISSUE THREAD

**Issue by bhargavkumar-65** [reportportal/reportportal#919]
**AzureDevOps Plugin*
...

Hi RP Team,
I am Using AzureDevops to Build my framework, I am using Reportportal and Kibana as my Reporting Tools, I Would love to have the Integration with AzureDevOps to track the Issues, Raise Bugs directly to AzureDevOps., And Update the Issue Comments from RP to AzureDevops.
Here is My project.: [](https://github.com/AutoInfra/AutoInfra)

**Comment by alexvaut** [comment]
+1 nice integration with azure devops project management would be great indeed ! Just creating a bug from a failing test would be a start.

**Comment by IvanKirianov** [comment]
Indeed! ADO extension for native integration in pipelines will be brill!

**Comment by Nils1970** [comment]
This would be highly desirable

**Comment by evjlobanova** [comment]
We are planning this integration this year:
1st phase - BTS like integration
2nd phase - native integrtion in pipeline

**Comment by hideintheclouds** [comment]
+1

**Comment by evjlobanova** [comment]
Azure BTS will be available in version 5.7.
5.7 release will be in the Feb 2022

**Comment by bhargavkumar-65** [comment]
@Nils1970  @hideintheclouds @IvanKirianov @Brrovko @alexvaut The plugin is now available and 5.7 Is released But i see some issue with AZDevops connection setup, raised a issue for the same. https://github.com/reportportal/plugin-bts-azure/issues/40

cc: @evjlobanova

**Comment by surajshenoy** [comment]
Anyone!!!! Any updates on integration of Azure DevOps pipelines with RP. I could see presently integration only available with Jenkins for CI/CD. Is there a plugin or task that is available to integrate the same with Azure pipelines ? Or will it going to be released in near future?

**Comment by MaryiaRymarchyk** [comment]
@surajshenoy  as for now, this is not in our plans for the near future. However, if it's something urgent that you would like to have,  we have two potential solutions:  1) We may discuss it as part of our paid services where we would custom-develop a plugin per your needs, or 2) You are welcome to contribute independently by developing the feature yourself and later sharing it with the entire community.
